### PR TITLE
Essential endpoints

### PR DESCRIPTION
**Pull request by Naazyon**

### CODE REVIEW

File: backend/routers/imageRouter.js [hunk 1]
```diff
@@ -16,8 +16,7 @@ const dirToZip = require('../../helpers/dirToZip')
 const annotateImage = require('../../helpers/annotateImage')
 const uuid = require('uuid/v4')
 
-const imageModel = require('../models/imageModel')
-const Image = require('mongoose').model(imageModel.modelName)
+const Image = require('mongoose').model('image')
```
Comment (Pixeladed) [inline backend/routers/imageRouter.js]: :(

File: backend/routers/projectRouter.js [hunk 2]
```diff
@@ -39,10 +41,10 @@ router.get('/', async (req, res) => {
 */
 router.post('/', async (req, res) => {
   try {
-    const { title } = req.body
+    const { title, apiKey } = req.body
 
     if (!title) return res.status(400).json({ msg: 'Missing project title.' })
```
Comment (Pixeladed) [inline backend/routers/projectRouter.js]: I've just realised that we dont have project titles. Would it be possible to remove this validation?

**Comment by Pixeladed** [comment]
I think we're also missing `@google-cloud/vision` in our `package.json`

**Comment by ekinbukulmez** [comment]
Yes I noticed that while I was working on the app last night. I may have added it in my pr yesterday but not sure

**Pull request merged**